{
  "contest": 118,
  "handle": "sorrynotsorry",
  "address": "0x6262f720679C04cc7f2a14c0c03486D0287c4853",
  "risk": "2",
  "title": "DoS Risk due to calls to external contracts inside a loop",
  "issueId": 210,
  "issueUrl": "https://github.com/code-423n4/2022-05-runes-findings/issues/210"
}